src/picknik_ur_mobile_config/LICENSE

```diff
@@ -0,0 +1,25 @@
+Redistribution and use in source and binary forms, with or without
```

Comment:
Since we're including stl files from the [ridgeback repository](https://github.com/ridgeback/ridgeback_robot): Can we add them + do we need to adjust our license?

Reply:
The ridgeback repo doesn't have a license it looks like.
And we are just adding it as a submodule without modifying any source, so I don't think we need to do anything license-wise.